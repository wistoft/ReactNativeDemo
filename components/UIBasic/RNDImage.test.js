import React from 'react';
import renderer from 'react-test-renderer';

import { Text } from 'react-native';

import RNDImage from "./RNDImage";

test('renders correctly', () => {
	const tree = renderer.create(
			<RNDImage 
					source={{uri:"TestURL"}}
					style={{width:100}} 
				>
			</RNDImage>
		);
	
		expect(tree).toMatchSnapshot();
});
