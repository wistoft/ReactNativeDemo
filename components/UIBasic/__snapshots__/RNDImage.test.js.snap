// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Image
  source={
    Object {
      "uri": "TestURL",
    }
  }
  style={
    Array [
      Object {
        "margin": 10,
        "marginBottom": 0,
      },
      Object {
        "width": 100,
      },
    ]
  }
/>
`;
